packages/generator-single-spa/src/react/templates/.babelrc

```diff
@@ -4,5 +4,9 @@
     "@babel/preset-react"
   ],
   "plugins": [
+    ["@babel/plugin-transform-runtime", {
```

Comment:
The main bundle savings that this creates is with the [helpers option](https://babeljs.io/docs/en/next/babel-plugin-transform-runtime.html#helpers) defaulting to true - no need to redefine `classCallCheck` multiple times within a single webpack bundle. I omitted that option from this config since it's default is what we want.